src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/SendDailyReportTask.java

```diff
@@ -48,15 +48,11 @@ public SendDailyReportTask(
         this.emailSender = emailSender;
         this.from = from;
 
-        if (recipients == null) {
-            this.recipients = new String[0];
+        if (recipients == null || recipients.length == 0) {
+            throw new RuntimeException("No recipients configured for reports");
```

Comment:
Tests missing ?

Reply:
should be covered with same tests i imagine

Reply:
The logic is not changed at all, check for zero length has been moved to the same line as check for null.